straxen/plugins/position_reconstruction.py

```diff
@@ -9,6 +9,36 @@
 from warnings import warn
 export, __all__ = strax.exporter()
 
+
+from straxen import URLConfig
+
+# TODO
+#  #############
+#  Temporary placed here, I don't want to mess with #817
+@URLConfig.register('tf')
+def open_neural_net(model_path: str, **kwargs):
+    # Nested import to reduce loading time of import straxen
+    import tensorflow as tf
+    if not os.path.exists(model_path):
+        raise FileNotFoundError(f'No file at {model_path}')
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        tar = tarfile.open(model_path, mode="r:gz")
+        tar.extractall(path=tmpdirname)
+        return tf.keras.models.load_model(tmpdirname)
+
+
+@export
+@URLConfig.register('download')
+def download(file_name: str) -> str:
+    """
+    Download single and return path to object. Different from
+    "resource://" is that we don't to open the file.
+    """
+    downloader = straxen.MongoDownloader()
+    return downloader.download_single(file_name)

```

Comment:
@WenzDaniel do you mean to add a keyword argument to the `resource://` protocol (or even the `straxen.get_resource` function) to return a path instead of bytes? I would be on board with that

Reply:
I meant `get_resources` itself, but resource protocol is fine too in case we do not want to do it in get_resources.

Reply:
@jmosbacher @WenzDaniel I don't think `get_resource` is the right place to be honest.

Is it okay for you if we can add it to `resource://`? This I think is a good idea.